dotnet-client-libraries/YamlDotNet/App/UseCases/JsonSupport.cs

```diff
@@ -0,0 +1,15 @@
+using YamlDotNet.Serialization;
+
+namespace App.UseCases;
+
+public static class JsonSupport
+{
+    public static string SerializeToJson(string yaml)
+    {
+        var deserializer = new DeserializerBuilder().Build();
+        var yamlObject = deserializer.Deserialize(yaml);
+        var serializer = new SerializerBuilder().JsonCompatible().Build();
+        var json = serializer.Serialize(yamlObject);
```

Comment:
This can be returned directly too, without an extra variable:

```suggestion
        return serializer.Serialize(yamlObject);
```

Reply:
@grantwinney Done